designsafe/static/scripts/data-depot/components/projects/publication-preview/publication-preview-hyb-sim.component.js

```diff
@@ -73,79 +70,52 @@ class PublicationPreviewHybSimCtrl {
                 });
                 child.setEntities(this.projectId, entities);
```

Comment:
Does this `setEntities` call still do anything?

Reply:
Yes, we add all associated entities to the child(file) of the listing in order to display the specific category and tags per category of each child(file).

Reply:
In what situation would `browser.listing.children` be used in the publication preview? Naively it looks like the listings are all handled through `populateListing`.

Reply:
When a user within the preview or published view of a project navigates into a directory we switch from the abstracted file listing `browser.listings` to the standard listing `browser.listing`. The children within these file paths can have tagged files as well.